models/endoscopic_inbody_classification/configs/inference_trt.json

```diff
@@ -4,7 +4,9 @@
         "$import json",
         "$import torch_tensorrt"
     ],
-    "handlers#0#_disabled_": true,
     "network_def": "$torch.jit.load(@bundle_root + '/models/model_trt.ts')",
-    "evaluator#amp": false
+    "evaluator#amp": false,
+    "initialize": [
+        "$monai.utils.set_determinism(seed=123)"
```

Comment:
May I ask why add the extra `set_determinism` function here? Do you have some difference results when running with TensorRT?

Reply:
Hi @binliunls , since we add `"$@checkpointloader(@evaluator)"` in initialize section in `inference.json`, it's not used for TRT, thus we should override initialize here.

Reply:
LGTM. Thanks.